File: src/main/java/catchla/yep/view/LocationContainerFrameLayout.java
package catchla.yep.view;

import android.content.Context;
import android.util.AttributeSet;

import com.commonsware.cwac.layouts.AspectLockedFrameLayout;

/**
 * Created by mariotaku on 16/1/30.
 */
public class LocationContainerFrameLayout extends AspectLockedFrameLayout {
    public LocationContainerFrameLayout(final Context context) {
        this(context, null);
    }

    public LocationContainerFrameLayout(final Context context, final AttributeSet attrs) {
        super(context, attrs);
        setAspectRatio(2.5);
    }
}

File: src/main/kotlin/catchla/yep/view/LocationContainerFrameLayout.kt
package catchla.yep.view

import android.content.Context
import android.util.AttributeSet

import com.commonsware.cwac.layouts.AspectLockedFrameLayout

/**
 * Created by mariotaku on 16/1/30.
 */
class LocationContainerFrameLayout @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null) : AspectLockedFrameLayout(context, attrs) {

    init {
        setAspectRatio(2.5)
    }
}
